### PR TITLE
Replace RCUTILS_ with RCLCPP_ for logging

### PR DESCRIPTION
**Pull request by JafarAbdi**
Fix: [#59](https://github.com/ros-controls/ros2_control/issues/59)

### CODE REVIEW

**Comment by JafarAbdi** [comment]
> However, the github action fail with some uncrustify errors:
> 

Running ament_uncrustify locally gives no error what could be the reason .?

I'll fix the other alpha sort comment today, Thanks!

**Comment by Karsten1987** [comment]
I suspect that you’re using eloquent but the current master branch is on par with the latest ros2 master and thus the uncrustify version used has changed. 

That is annoying, but for this particular problem I believe you don’t have to build your ros2 version from scratch but solely rely on the linked github actions or travis.

File: controller_manager/src/controller_manager.cpp [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 #include "lifecycle_msgs/msg/state.hpp"
 
-#include "rcutils/logging_macros.h"
```
Comment (Karsten1987) [inline controller_manager/src/controller_manager.cpp]: With `rcutils` not being used here, we can also lift the dependency to `rcutils` in the `package.xml`, can't we?

File: hardware_interface/CMakeLists.txt [hunk 2]
```diff
@@ -11,6 +11,7 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 endif()
 
 find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
 find_package(rcpputils REQUIRED)
 find_package(rcutils REQUIRED)
```
Comment (Karsten1987) [inline hardware_interface/CMakeLists.txt]: is that still needed?

File: test_robot_hardware/include/test_robot_hardware/test_robot_hardware.hpp [hunk 3]
```diff
@@ -17,6 +17,8 @@
 
 #include <string>
 
+#include "rclcpp/rclcpp.hpp"
```
Comment (Karsten1987) [inline test_robot_hardware/include/test_robot_hardware/test_robot_hardware.hpp]: alphabetical sort

**Comment by Karsten1987** [comment]
@JafarAbdi Any updates on this?

**Comment by JafarAbdi** [comment]
@Karsten1987 Sorry, I just saw your last review, I addressed the latest review and pushed the changes

File: test_robot_hardware/package.xml [hunk 4]
```diff
@@ -10,9 +10,11 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
 
   <build_depend>hardware_interface</build_depend>
+  <build_depend>rclcpp</build_depend>
   <build_depend>rcutils</build_depend>
```
Comment (bmagyar) [inline test_robot_hardware/package.xml]: can this go?

**Pull request merged**